plugins/module_utils/runner.py

```diff
@@ -23,11 +23,20 @@
     OPENAPI_SCHEMA_IMPORT_ERROR = None
 
 from .api_module import FlightctlAPIModule
-from .constants import Kind
+from .constants import ResourceType
 from .exceptions import FlightctlException, ValidationException
 from .inputs import ApprovalInput
 from .resources import create_definitions
 
+try:
```

Comment:
If anybody is wondering about the try / except handling of external dependencies this is an Ansible pattern, more info [here](https://github.com/ansible/ansible/issues/78346#issuecomment-1939191550)